peer_heap.go

```diff
@@ -44,7 +45,7 @@ func (ph *peerHeap) Less(i, j int) bool {
 	if ph.peerScores[i].score == ph.peerScores[j].score {
 		return ph.peerScores[i].order < ph.peerScores[j].order
```

Comment:
That's odd - you're right, I'd expect this to be prevent compilation. Weirdly, it doesn't: https://travis-ci.org/uber/tchannel-go/jobs/117770780.

I'll look into whether we need the atomics here.